.rubocop.yml

```diff
@@ -1,39 +1,12 @@
-require:
-  - rubocop-performance
-  - rubocop-rails
-  - rubocop-minitest
-
-AllCops:
-  TargetRubyVersion: 2.7
-  TargetRailsVersion: 6.1
-  Exclude:
-    - 'node_modules/**/*'
-    - 'vendor/**/*'
-
-Layout/ArgumentAlignment:
-  EnforcedStyle: with_fixed_indentation
-  IndentationWidth: 2
-
-Layout/EmptyLineAfterGuardClause:
-  Enabled: false
+inherit_gem:
+  theforeman-rubocop:
+    - default.yml
 
 Layout/LineLength:
   Enabled: 111 # TODO: discuss and set this
```

Comment:
I'd also drop this an the other 2. Just stick to `theforeman-rubocop` defaults. If we think it should be different, then that's the place to change it IMHO.